src/ray/raylet/node_manager.cc

```diff
@@ -900,11 +904,7 @@ void NodeManager::ResourceCreateUpdated(const NodeID &node_id,
                                                         new_resource_capacity);
   }
   RAY_LOG(DEBUG) << "[ResourceCreateUpdated] Updated cluster_resource_map.";
-
-  if (node_id == self_node_id_) {
```

Comment:
So this logic becomes the reverse of what it is before. Only non-local nodes trigger scheduling now. Why it is desirable and maybe add a comment?

Reply:
yes, that's right. ok let me add some comments.